stdlib/2.7/__builtin__.pyi

```diff
@@ -913,8 +1003,8 @@ class file(BinaryIO):
     def __init__(self, file: unicode, mode: str = 'r', buffering: int = ...) -> None: ...
```

Comment:
Hm, maybe we can commandeer 'file' (a PY2-only class) to be the subclass of IO[str] I wished for above.